### PR TITLE
feat(agora): migration About and related components (AG-1545)

### PR DESCRIPTION
**Pull request by sagely1**
## Description

Migration of About Component and the following dependent components:
- loading-icon
- Wiki component

Updated common stylesheets that will be reused among components

Added Mock Service Worker package to mock API calls

## Related Issue

- [AG-1545](https://sagebionetworks.jira.com/browse/AG-1545)

## Changelog

- Add loading-icon component (used in loading animations)
- Add wiki component (gets content in about page from Synapse)
- Added Mock Service Worker package and configured wiki tests to mock api calls
   
## Testing
`nx test agora-about`
`nx test agora-wiki`

## Expected output
Navigate to:
http://localhost:4200/about

image

[AG-1545]: https://sagebionetworks.jira.com/browse/AG-1545?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: jest.config.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
-const { getJestProjects } = require('@nx/jest');
```
Comment (sagely1) [inline jest.config.ts:1]: updating to match the latest config:
https://nx.dev/nx-api/jest#configurations

File: libs/agora/wiki/jest.config.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+export default {
+  displayName: 'agora-wiki',
+  preset: '../../../jest.preset.js',
+  setupFilesAfterEnv: ['<rootDir>/src/test-setup.ts'],
+  globals: {
+    Request,
+    Response,
+    TextEncoder,
+    TextDecoder,
+  },
```
Comment (sagely1) [inline libs/agora/wiki/jest.config.ts:11]: Needed to support msw mocking

File: libs/agora/wiki/src/test-setup.ts [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  // Reset the request handlers between each test.
+  // This way the handlers we add on a per-test basis
+  // do not leak to other, irrelevant tests.
+  server.resetHandlers();
+});
+
+afterAll(() => {
+  // Finally, disable API mocking after the tests are done.
+  server.close();
+});
```
Comment (sagely1) [inline libs/agora/wiki/src/test-setup.ts:20]: This block adds the ability to mock the api calls.  Default api behavior is defined in libs/agora/testing...

**Review by tschaffter — APPROVED** [review]
LGTM

**Review by tschaffter — CHANGES_REQUESTED** [review]
Aside a task that fails in the CI workflow, LGTM.

**Review by tschaffter — APPROVED** [review]

**Pull request merged**